ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/command/editor/page/goal/GoalPage.java

```diff
@@ -60,13 +61,10 @@ public IsWidget getView() {
 
     @Override
     protected void initialize() {
-        final String goalId = editedCommand.getGoal();
-        final CommandGoal goal = goalRegistry.getGoalForId(goalId);
-
-        goalInitial = goal.getId();
+        goalInitial = editedCommand.getGoal();
```

Comment:
Shouldn't it be `initialGoal`? I'm a bit off the context, but this name seems quite odd to me.